internal/storage/bucket_handle_test.go

```diff
@@ -1319,13 +1321,46 @@ func (testSuite *BucketHandleTest) TestGetFolderWhenFolderDoesNotExistsForHierar
 func (testSuite *BucketHandleTest) TestRenameFolderWithError() {
 	ctx := context.Background()
 	mockClient := new(MockStorageControlClient)
-	mockClient.On("RenameFolder", mock.Anything, &controlpb.RenameFolderRequest{Name: "projects/_/buckets/" + TestBucketName + "/folders/" + TestObjectName, DestinationFolderId: TestRenameFolder}, mock.Anything).
+	mockClient.On("RenameFolder", mock.Anything, &controlpb.RenameFolderRequest{Name: fmt.Sprintf(FullFolderPathHNS, TestBucketName, TestFolderName), DestinationFolderId: TestRenameFolder}, mock.Anything).
 		Return(nil, errors.New("mock error"))
 	testSuite.bucketHandle.controlClient = mockClient
 	testSuite.bucketHandle.bucketType = gcs.Hierarchical
 
-	_, err := testSuite.bucketHandle.RenameFolder(ctx, TestObjectName, TestRenameFolder)
+	_, err := testSuite.bucketHandle.RenameFolder(ctx, TestFolderName, TestRenameFolder)
 
 	mockClient.AssertExpectations(testSuite.T())
 	assert.NotNil(testSuite.T(), err)
 }
+
+func (testSuite *BucketHandleTest) TestCreateFolderWithError() {
+	mockClient := new(MockStorageControlClient)
+	mockClient.On("CreateFolder", context.Background(), &controlpb.CreateFolderRequest{Parent: fmt.Sprintf(FullBucketPathHNS, TestBucketName),
+		FolderId: TestFolderName}, mock.Anything).
```

Comment:
can you extract this construction of CreateFolderRequest into seperate line and assign it to a local variable. 
go/go-style/decisions#func-formatting

Reply:
Done